src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4783,6 +4783,60 @@ void M()
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        [WorkItem(25990, "https://github.com/dotnet/roslyn/issues/25990")]
+        public async Task TestWithLineBreak_WithSimpleComment()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        int x = // start [| comment
+            5 * 2 |]
```

Comment:
this case seems weird... i'm not sure this would be something i would expect to work.  if the user has a selection within a comment... offering a refactoring on code seems undesirable.

Reply:
Note: do we have a test for the user scenario?  i think their's was specifically:

```c#
        int x = 
[|            5 * 2 
|]                ;
```

i.e. they selected the line by going to the start of the line they wants to select, and hit 'shift-down'.

Reply:
is there a test for this?

Reply:
Thanks for the ping. Just pushed such a test.